src/components/common/Extensions/UserHubButton/UserHubButton.tsx

```diff
@@ -84,40 +76,46 @@ const UserHubButton: FC = () => {
           {
             name: 'offset',
             options: {
-              offset: popperTooltipOffset,
+              offset: isMobile ? [0, 1] : [0, 8],
```

Comment:
Ufff.

I know this was the "way" to make this work, but god damn these might as well be "magic" numbers at this point :)